namespaces/live-1.cloud-platform.service.justice.gov.uk/manage-intelligence-dev/00-namespace.yaml

```diff
@@ -0,0 +1,12 @@
+apiVersion: v1
+kind: Namespace
+metadata:
+  name: manage-intelligence-dev
+  labels:
+    cloud-platform.justice.gov.uk/is-production: "false"
+    cloud-platform.justice.gov.uk/environment-name: "dev"
+  annotations:
+    cloud-platform.justice.gov.uk/business-unit: "HMPPS"
+    cloud-platform.justice.gov.uk/application: "manage-intelligence"
```

Comment:
Please add a slack channel (minus the leading `#`) that we can use if we need to contact the team which owns this namespace.